### PR TITLE
replace mongodb with nedb

### PR DESCRIPTION
**Pull request by frankobe**

### CODE REVIEW

File: src/db/nedb.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+module.exports = async () => {
+  await Promise.all(dbPromises);
+
+  return {
+    Topics: topics,
+    Oracles: oracles,
+    Votes: votes,
+    Blocks: blocks
+  };
+}
```
Comment (dwalintukan) [inline src/db/nedb.js]: add EOF line

File: src/db/nedb.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+topics = datastore({filename: 'topics.db', autoload: true});
+oracles = datastore({filename: 'oracles.db', autoload: true});
+votes = datastore({filename: 'votes.db', autoload: true});
+blocks = datastore({filename: 'blocks.db', autoload: true});
+
+dbPromises = [topics, oracles, votes, blocks]
+
+module.exports = async () => {
+  await Promise.all(dbPromises);
```
Comment (dwalintukan) [inline src/db/nedb.js]: Should wrap this in a try catch block so its not a unhandled promise.

File: src/index.js [hunk 3]
```diff
@@ -10,7 +10,8 @@ const {graphqlExpress, graphiqlExpress} = require('apollo-server-express');
 
 const schema = require('./schema');
 
-const connectDB = require('./db');
+// const connectDB = require('./db');
```
Comment (dwalintukan) [inline src/index.js]: remove if not needed?

File: src/sync/index.js [hunk 4]
```diff
@@ -413,7 +415,7 @@ async function sync(db){
 async function updateOraclesPassedEndBlock(currentBlockChainHeight, db, resolve){
   // all central & decentral oracles with VOTING status and endBlock less than currentBlockChainHeight
   try {
-    await db.Oracles.findAndModify({endBlock: {$lt:currentBlockChainHeight}, status: 'VOTING'}, [], {$set: {status:'WAITRESULT'}}, {});
+    await db.Oracles.update({endBlock: {$lt:currentBlockChainHeight}, status: 'VOTING'}, {$set: {status:'WAITRESULT'}}, {multi: true});
```
Comment (dwalintukan) [inline src/sync/index.js]: why does this one have {multi: true} and some others dont?

Comment (frankobe) [reply src/sync/index.js]: without `{multi: true}`, only one row returned from query will be updated

File: src/db/nedb.js (hunk 1 above)
Comment (frankobe) [reply src/db/nedb.js]: done

**Pull request merged**